pkg/gather/clusterconfig/0_gatherer.go

```diff
@@ -95,15 +104,19 @@ var gatherFunctions = map[string]gathering{
 	"olm_operators":                     failable(GatherOLMOperators),
 }
 
-var startTime time.Time
-
 // New creates new Gatherer
-func New(gatherKubeConfig *rest.Config, gatherProtoKubeConfig *rest.Config, metricsGatherKubeConfig *rest.Config) *Gatherer {
-	startTime = time.Now()
+func New(
```

Comment:
It is just a suggestion, but you can simplify this one by using:

```
func New(gatherKubeConfig, gatherProtoKubeConfig, metricsGatherKubeConfig *rest.Config, anonymizer *anonymization.Anonymizer)
```

Reply:
done